### PR TITLE
Docker CP instead of rsync builds in project/.gitlab-ci-local

### PR DESCRIPTION
**Pull request by firecow**

### CODE REVIEW

File: src/job.ts [hunk 1]
```diff
@@ -248,24 +251,20 @@ export class Job {
...
-            }
-            preCmd += `"\n`;
-            await Utils.spawn(preCmd, this.cwd);
-            const endTime = process.hrtime(time);
-            process.stdout.write(`${this.getJobNameString()} ${magentaBright(`rsync to build folder`)} in ${magenta(prettyHrtime(endTime))}\n`);
+            time = process.hrtime();
+            process.stdout.write(`${jobNameStr} ${magentaBright(`pulling`)} ${this.image}\n`);
+            let pullCmd = ``;
+            pullCmd += `docker image ls --format '{{.Repository}}:{{.Tag}}' | grep -E '^${this.image}$'\n`
+            pullCmd += `if [ "$?" -ne 0 ]; then\n`
```
Comment (ZobairQ) [inline src/job.ts:258]: wouldn't this look better as one expression?

Comment (firecow) [reply src/job.ts:258]: Sure, if we don't care about line length 😃
And I don't so will change

**Review by FnuGk — APPROVED** [review]
LGTM

File: src/job.ts (hunk 1 above)
Comment (firecow) [reply src/job.ts:258]: Wait, we cannot put all this on one line it will be way too long

Comment (firecow) [reply src/job.ts:258]:
```
            pullCmd += `docker image ls --format '{{.Repository}}:{{.Tag}}' | grep -E '^${this.image}$'\n`
            pullCmd += `if [ "$?" -ne 0 ]; then\n`
            pullCmd += `\techo "Pulling ${this.image}"\n`
            pullCmd += `\tdocker pull ${this.image}\n`
            pullCmd += `fi\n`
```

**Review by ZobairQ — APPROVED** [review]
LGTM

**Pull request merged**